eng/testing/workloads-browser.targets

```diff
@@ -47,7 +47,8 @@
   <Target Name="_GetRuntimePackNuGetsToBuild" Condition="'$(WasmSkipMissingRuntimePackBuild)' != 'true'" Returns="@(_NuGetsToBuild)">
     <Error Condition="'$(RIDForWorkload)' == ''" Text="$(RIDForWorkload) is unset" />
     <PropertyGroup>
-      <_DefaultBuildVariant Condition="'$(MonoWasmBuildVariant)' == 'multithread'">.multithread.</_DefaultBuildVariant>
+      <_BuildVariant Condition="'$(WasmEnableThreads)' == 'true'">multithread"</_BuildVariant>
```

Comment:
this is part which I don't understand well.
It was item in list and now it's not.
Are we ever building both variants ?

Reply:
We used to keep build variants in a list, instead of property - list implies that there might be more than one variant being built at the same run. When we look at the flow, though we can see that it's not feasible - the addition was controlled by `WasmEnableThreads` and only one value was added: either "." or ".multithread.", so we have 2 exclusive options.

Reply:
I think I have an answer. Historically there were 3 options: multithread, perftrace, empty. And they were not mutually exclusive, see:
https://github.com/ilonatommy/runtime/commit/d7d0e421d77a7c95894a79262a7528fd825e25b0